crates/astria-sequencer/src/app.rs

```diff
@@ -293,13 +297,35 @@ impl App {
                 continue;
             };
 
-            // store transaction execution result, indexed by tx hash
             let tx_hash = Sha256::digest(&tx);
+
+            // Sum up the total bytes in sequence actions
+            let mut sequenced_data_size = 0;
+            for action in signed_tx.clone().into_unsigned().actions {
```

Comment:
No need to clone here:
```suggestion
            for action in &signed_tx.unsigned_transaction().actions {
```
If you want to be fancy:

```
let total_len = signed_tx
    .unsigned_transaction()
    .iter()
    .filter_map(UnsignedTransaction::as_sequence)
    .fold(0usize, |acc, seq| acc + seq.data.len());
```